internal/codespaces/api/api.go

```diff
@@ -537,6 +537,7 @@ func (a *API) GetCodespacesMachines(ctx context.Context, repoID int, branch, loc
 	q := req.URL.Query()
 	q.Add("location", location)
 	q.Add("ref", branch)
+	q.Add("devcontainer_path", devcontainerPath)
```

Comment:
Just confirming, if the is empty string the call is OK?

Reply:
If the server is fine with it than it should be fine

Reply:
Yup empty string is fine as it's an optional parameter